src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs

```diff
@@ -34,7 +34,7 @@ public override async Task<IEnumerable<IPackageSearchMetadata>> SearchAsync(stri
             return searchResults;
         }
 
-        private static IEnumerable<VersionInfo> GetVersions(PackageSearchMetadata metadata, SearchFilter filter)
+        protected static IEnumerable<VersionInfo> GetVersions(PackageSearchMetadata metadata, SearchFilter filter)
```

Comment:
I don't imagine you need this change anymore.
You can revert it by running the following from the command line

```console
git checkout dev -- src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs 
```

Reply:
Thanks. I reverted the change.